GVFS/GVFS.Service/Mac/GVFSMountProcess.cs

```diff
@@ -0,0 +1,60 @@
+﻿using GVFS.Common;

```

Comment:
Removed `PrjFlt` related code from here. This class is referenced from `RepoRegistration`.

Reply:
Removed all Stubs

Reply:
Removed all Stubs